.github/workflows/ci.yml

```diff
@@ -10,6 +10,9 @@ on:
 
 jobs:
   test:
+    permissions:
+      # Needed to access the workflow's OIDC identity.
+      id-token: write
```

Comment:
N.B.: I added this so that the `test_sign_rekor_entry_consistent` test can use the GitHub ambient OIDC credentials.

Reply:
Do we need additional markers to mark tests as 'interactive'? I would expect these to be somewhat annoying when trying to run the test suite locally without ambient credentials.

Reply:
Yeah, we could add an interactive marker. Right now I have it set up such that the tests are skipped locally if no ambient credential is found, but we could have it do the normal interactive credential discovery instead.

Reply:
Oh, that's fine as well, I wasn't aware that's the case.